test/integration/src/test/java/io/pravega/test/integration/BatchClientTest.java

```diff
@@ -173,6 +178,38 @@ public void testBatchClient() throws Exception {
         ArrayList<String> dataAtOffset = Lists.newArrayList(seg0Iterator);
         assertEquals(1, dataAtOffset.size());
         assertEquals(DATA_OF_SIZE_30, dataAtOffset.get(0));
+
+        //Test with Stream Truncation.
```

Comment:
Please, check that comments comply with the style rules (e.g., starts by white space, blank line prior comment line).

Reply:
corrected it.